syntax = "proto3";
package Responses;

// Response after server loads a new file
message FileLoadResponse {
    bool success = 1;
    string filename = 2;
    int32 image_width = 3;
    int32 image_height = 4;
    int32 image_depth = 5;
}